tasks/24_index-to-insert/index-to-insert.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * getIndexToInsert - returns the lowest index at which
+ * a value should be inserted into an array once it has been sorted
+ *
+ * @param {Array<Number>} arr
+ * @param {Number} num
+ * @return {Number}
+ */
+export function getIndexToInsert(arr, num) {
+  const sortedArray = arr.sort((a, b) => a - b);
```

Comment:
I believe it works this way by default so no need to pass function into sort

Reply:
By default sort() compare elements of the array as String. If we want correct sort array of numbers we should pass function into sort.